nameservice/nameservice/config.yml

```diff
@@ -1,9 +1,9 @@
 version: 1
 accounts:
-  - name: jack
```

Comment:
In previous versions of the tutorial we use jack/alice as opposed to user1/user2, so [21-run-rest](http://localhost:8080/nameservice/tutorial/21-run-rest.html) should also be updated.

Reply:
Thanks. Addressed 👍